assets/js/common/mozilla-utils.js

```diff
@@ -40,60 +40,72 @@ if (typeof Mozilla === 'undefined') {
 
     // attach an event to all the download buttons to trigger the special
     // ie functionality if on ie
-    Utils.initDownloadLinks = function() {
+    Utils.initDownloadLinks = function () {
         $('#submit-button').hide();
-        $('.download-link').each(function() {
+        $('.download-link').each(function () {
             var $el = $(this);
-            $el.click(function(e) {
-                e.preventDefault();
-                window.Mozilla.Donation.DisplayAmountForm();
-                // Utils.triggerIEDownload($el.data('direct-link'));
+            $el.click(function (e) {
+                if (window.ABTest.IsInFundraiseUpBucket()) {
```

Comment:
It feels like a bunch of this stuff should be in a function. Also, lets leave the AB test off for now because it seems like it'll be a bit annoying while we're just prototyping the new flow itself.